modules/module_catalog/Ed-Fi/notebook/Refine_EdFi.ipynb

```diff
@@ -117,7 +117,7 @@
           "output_type": "display_data",
           "data": {
             "application/vnd.livy.statement-meta+json": {
-              "spark_pool": "spark3p1sm",
+              "spark_pool": "spark3p3sm",
```

Comment:
Please check with David on the spark node reqs for Ed-Fi.

Reply:
I can also exclude all other spark_pool updates and just keep PR scope to cover OEA frame_work and MS Education Insight spark_pool changes to it?